### ISSUE TITLE
ExecuteSwapRequest improvements

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#734]
add `result: completed` / `failed`. This won't necessarily be used programmatically in the end, but makes the situation much clearer for engineers integrating & testing `xud`. Feedback from our test integration.

```
result: "completed"
order_id: "631145c0-f901-11e8-b518-5fd626b71b77"
pair_id: "LTC/BTC"
quantity: 0.001
r_hash: "aad8fae44138922fc52b1fca949c7f75294fa5f8345418634d23ba4d961b5b9f"
amount_received: 100000
amount_sent: 110000
peer_pub_key: "03fd337659e99e628d0487e4f87acf93e353db06f754dccc402f2de1b857a319d0"
currency_received: "LTC"
currency_sent: "BTC"
```

**Comment by ImmanuelSegol** [comment]
@kilrau Did you mean I should add this to the `SwapResult` to indicate if the swap was successful  ?

**Comment by kilrau** [comment]
yep

**Comment by sangaman** [comment]
I think that this will be effectively resolved by #787, which distinguishes between swap successes and failures by naming the fields `SwapSuccess` or `SwapSuccesses` and `SwapFailure`.

**Comment by kilrau** [comment]
Ok, good to close this one then. Naturally you'll get rewarded for your work nevertheless @ImmanuelSegol
